lib/Diagnostics.php

```diff
@@ -76,19 +78,29 @@ public function __construct(IConfig $config, IUserSession $session) {
 	}
 	
 	/**
-	 * @param string $uids - e.g. "["admin","user1000"]" in JSON format
+	 * Function accepts as parameters JSON formated string e.g. ["{\"id\":\"test\",\"displayname\":\"User, Test\"}", ..].
+	 * JSON contains array of JSON objects with following keys:
+	 *  id - e.g. test
+	 *  displayname - e.g.
+	 *
+	 * @param string $userData
 	 */
-	public function setDiagnosticForUsers($uids) {
-		$this->config->setAppValue('diagnostics', 'diagnoseUsers', $uids);
-		$this->diagnosticForUsers = json_decode($uids, true);
+	public function setDiagnosticForUsers($userData) {
+		$this->config->setAppValue('diagnostics', 'diagnosedUsers', $userData);
```

Comment:
This might mean, after the update, users of beta version will need to select users again, also there would be dummy record in the DB, unless cleaned @PVince81 
![selection_186](https://user-images.githubusercontent.com/13368647/29998163-a07fd76a-9024-11e7-93e6-d9ec04f25ba2.png)